src/bindings/c/tests/ov_preprocess_test.cpp

```diff
@@ -132,6 +132,20 @@ TEST_F(ov_preprocess_test, ov_preprocess_preprocess_steps_scale) {
     OV_EXPECT_OK(ov_preprocess_preprocess_steps_scale(input_process, 2.0f));
 }
 
+TEST_F(ov_preprocess_test, ov_preprocess_preprocess_steps_scale_multi_channels) {
+    OV_EXPECT_OK(ov_preprocess_prepostprocessor_create(model, &preprocess));
+    EXPECT_NE(nullptr, preprocess);
+
+    OV_EXPECT_OK(ov_preprocess_prepostprocessor_get_input_info_by_index(preprocess, 0, &input_info));
+    EXPECT_NE(nullptr, input_info);
+
+    OV_EXPECT_OK(ov_preprocess_input_info_get_preprocess_steps(input_info, &input_process));
+    EXPECT_NE(nullptr, input_process);
+
+    float values[4] = {2.0f, 2.0f, 2.0f, 2.0f};
+    OV_EXPECT_OK(ov_preprocess_preprocess_steps_scale_multi_channels(input_process, values, 4));
```

Comment:
should we check that `ppp.build()` finished?

Reply:
yes, it doesn't trigger actual preprocessing task, because such tests are only used to validate whether new wrapper APIs have any problem, not focused on triggering actual preprocessing. Actually, we have following test cases `ov_preprocess_prepostprocessor_build_apply` and `ov_preprocess_prepostprocessor_for_nv12_input` to validate whether actual preprocessing can work with corresponding preprocess preparation parameters.
Anyhow, let's  apply `ppp.build()` for more test items.